utilities/test_suite/rpp_test_suite_audio.h

```diff
@@ -316,4 +316,46 @@ void verify_non_silent_region_detection(int *detectedIndex, int *detectionLength
         qaResults << status << std::endl;
         qaResults.close();
     }
+}
+
+inline Rpp32f sinc(Rpp32f x)
```

Comment:
when we enable RPP HIP audio, we need to remove these from here and MIVisionX. To do that you can use a new RPP version and check that